### ISSUE TITLE
User Wizard does not accept "admin@localhost" as a valid email

### ISSUE THREAD

**Issue by Ricket** [datatypevoid/vulgar#19]
- *_I'm submitting a ... *_
  
  [x] bug report
  
  [ ] feature request
  
  [ ] question about the decisions made in the repository
- **Do you want to request a _feature_ or report a _bug_?**
  report a _bug_
- **What is the current behavior?**
  After install you can summon the "User Wizard". He asks for an email address to link to the admin account. When I type `admin@localhost`, he says "Invalid input for email".
- **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** via https://plnkr.co, https://codepen.io, or similar.
  `vulgar init foo` and then opt to summon the User Wizard and type email `admin@localhost`
- **What is the expected behavior?**
  `admin@localhost` is a valid email address. It should be accepted.
- **What is the motivation / use case for changing the behavior?**
  Bad email address validation

**Comment by datatypevoid** [comment]
Yeah this is a known bug on the cli. Just needs an improved regex. It will be addressed as soon as the development branch of this project is merged with master.
